boxsdk/client/client.py

```diff
@@ -317,6 +317,39 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=100, offset=0, fields=None):
+        """
+        Get the user's recently accessed items.
+
+        :param: limit
+            The maximum number of items to return
+        :type: limit
+            `int`
+        :param offset:
+            The index at which to start returning items.
+        :type offset:
+            `int`
+        :param fields:
+            List of fields to request.
+        :type fields:
+            `Iterable` of `unicode`
+        :returns:
+            A list of the user's recently accessed items.
+        :rtype:
+            `list` of :class:`Item`
+        """
+        url = '{0}/recent_items'.format(API.BASE_API_URL)
```

Comment:
Everything else in the file follows the format used here. Should I still change it?